MapboxNavigation/NavigationViewController.swift

```diff
@@ -649,7 +658,9 @@ extension NavigationViewController: RouteControllerDelegate {
                 guard let self else {
                     return
                 }
-                self.delegate?.navigationViewControllerDidFinishRouting?(self)
+                self.mapViewController.hideEndOfRoute { _ in
+                    self.delegate?.navigationViewControllerDidFinishRouting?(self)
+                }
```

Comment:
I've ensured that this is called regardless of `showsEndOfRouteFeedback` Is set to true or false.